data-pipeline/derived-de-duplication/src/main/java/org/ekstep/ep/samza/domain/Event.java

```diff
@@ -48,13 +48,5 @@ public void markFailure(String error, DeDuplicationConfig config) {
         telemetry.add("metadata.src", config.jobName());
     }
 
```

Comment:
Any reason for the following code to be removed?

Reply:
We were updating the default channel in the TelemetryValidator job. I think this code has been duplicated in almost all jobs to update the default channel. I think we don't need the default channel updates in every job.